keyboards/ploopyco/mouse/mouse.c

```diff
@@ -167,11 +179,19 @@ bool process_record_kb(uint16_t keycode, keyrecord_t* record) {
     }
 
     if (keycode == DRAG_SCROLL) {
+#if PLOOPY_DRAGSCROLL_MOMENTARY==1
+        is_drag_scroll ^= 1;
```

Comment:
While it works out the same, this should be "safer". 
```suggestion
        is_drag_scroll = record->event.pressed;
```